ingestion/create_deploy.sh

```diff
@@ -154,20 +160,36 @@ create_commit_log() {
   local to_commit=$4
   
   touch "$log_path"
-  git --git-dir="$git_repo_loc/.git" log --pretty=format:'%H %ad' --date=iso "$from_commit".."$to_commit" > "$log_path"
+  git --git-dir="$git_repo_loc/.git" log --pretty=format:'%H %at000' --date=iso "$from_commit".."$to_commit" > "$log_path"
```

Comment:
Was the change to the format because the COMMIT_OVERRIDE looks more like this if it's present?

Reply:
yeah, the %ad format emitted a string like '2024-10-02 09:12:30 -0400' and %at emits a unix timestamp (in seconds), but jenkins gives us a timestamp in milliseconds so to have a common format I just used %at and added 000 milliseconds.